eradiate/scenes/biosphere/_discrete.py

```diff
@@ -202,6 +221,14 @@ class LeafCloudParams:
         type="float",
     )
 
+    position_offset = documented(
+        pinttr.ib(default=0.0 * ureg.m, units=ucc.deferred("length")),
+        doc="z-offset for leaf positions. Used for trees with trunks.\n\n"
+            "Unit-enabled field (default: ucc[length]).",
+        type="float",
+        default="0.0 m",
+    )
```

Comment:
The overall idea is that those leaf clouds are generated in a reference frame and then positioned by the user. I think you can do without this field: just apply the shift after generating the cloud.